### PR TITLE
get_group works now :boom:

### PR DESCRIPTION
**Pull request by lrosa007**

### CODE REVIEW

**Comment by wmjbyatt** [comment]
Let me take a look at this a bit more serious before it gets accepted

File: migo.js [hunk 1]
```diff
@@ -61,13 +66,17 @@ class Board {
   }
 
   test_get_group(){
```
Comment (wmjbyatt) [inline migo.js]: I think I'm going to extract this into a real testing library and encourage contributors to write tests

**Comment by wmjbyatt** [comment]
I see that you changed the es6 to a js extension. I talked to Carlos and he seemed to appreciate the es6 extension, do we have a reason to go back to js?

**Comment by lrosa007** [comment]
No, I was just renamed it to see if it would work in chrome and then just didn't change it back. I'll change it back.

File: app.html [hunk 2]
```diff
@@ -18,8 +18,8 @@
       var canvas = document.getElementById('board');
       var board = new Board();
 
-      canvas.width = 600;
-      canvas.height = 600;
+      canvas.width = board.BOARD_SIZE;
+      canvas.height = board.BOARD_SIZE;
```
Comment (lrosa007) [inline app.html]: ah I see. I did that cause I thought I remembered the interpreter being cool with it.

Comment (wmjbyatt) [inline app.html]: Yeah, one of the many, MANY annoying things about JS in the client: that value would simply be NULL, which just doesn't bother raising an exception. So it's a silent fail.

Comment (carias8) [inline app.html]: What test lib do you have in mind?

Comment (wmjbyatt) [inline app.html]: I'm just now seeing this, sorry I'm still getting used to project management via GitHub. Anyway, Mocha with Chai expectations is, in my experience, the easiest way to really get moving fast with a robust testing library. I've only used that with command-line runners before, so this use of a browser-based runner is an experiment for me. A command-line runner would still work here, we'd just have to mock the DOM, which is kind of a pain in the ass, and we'd probably want to mock for each test, which means maintaining parallelisms (unless we move to React and start loading component, which would be fucking cool but a) overkill and b) pretty sure React doesn't play nice with Canvas yet).

Right now there's definitely a weakness in that `test_runner.html` has to be kept up to date with `app.html`. But I think we may have answers for that once the server-side gets some work done. We'll see.

Comment (lrosa007) [inline app.html]: Using react would be cool. My rebuttals:
a) fuck it why not make it baller, lets match the overkill
b) toss out canvas?!?

That being said and all jokes aside. I'm pretty open as far as the direction of the tools goes.

Comment (wmjbyatt) [inline app.html]: I genuinely think that for this application Canvas makes WAY more sense. It's simpler and a LOT faster. Not that speed is really critical in this application, but if you get faster for free out of simpler, why the hell not? I also THINK that Canvas will be easier to scale to arbitrary board sizes, although I may have just not thought through the React way to do that yet.

That having been said, I do think as this goes from being a bare Goban to a robust web application, React will become the natural way to build out the rest of the interface. Like a chat window or whatever. And at that point the Canvas can become a React component itself, which might make encapsulation cleaner.

Comment (lrosa007) [inline app.html]: for sure

**Pull request merged**